src/main/java/org/example/Person.java

```diff
@@ -19,6 +19,7 @@
     private  List<Person> siblings = new ArrayList<>();
     private List<Person> children = new ArrayList<>();
     private List<Pet> pets = new ArrayList<>();
```

Comment:
Mooi gedaan. Deze oplossing noemde Mark in de laatste huiswerkklas, maar staat niet in de uitwerkingen. Ik vind het een slimme oplossing.

Reply:
Dat was ook in de klas:)